spec/lib/miq_automation_engine/miq_ae_domain_spec.rb

```diff
@@ -3,12 +3,14 @@
 
 describe MiqAeDomain do
   before do
+    EvmSpecHelper.local_guid_miq_server_zone
+    @root_tenant = Tenant.root_tenant
     setup_model
   end
 
   def setup_model
     yaml_file = File.join(File.dirname(__FILE__), 'data', 'domain_test.yaml')
-    import_options = {'yaml_file' => yaml_file, 'preview' => false, 'domain' => '*'}
+    import_options = {'yaml_file' => yaml_file, 'preview' => false, 'domain' => '*', 'tenant_id' => @root_tenant.id}
```

Comment:
@kbrock 
When we do an import into the database we are not using the Factories since the import code builds the domain,namespaces and classes.